schemas/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+<!-- Learn how to maintain this file at https://github.com/WordPress/gutenberg/tree/HEAD/packages#maintaining-changelogs. -->
+
```

Comment:
I guess we can keep the changelog file to leave some notes when there are some deprecations added.

Reply:
👍 That was my thought, nice to have to document any changes.